worlds/sc2/ItemDescriptions.py

```diff
@@ -672,99 +672,102 @@ def get_resource_efficiency_desc(item_name: str) -> str:
     ItemNames.DESTROYER_REFORGED_BLOODSHARD_CORE: "When fully charged, the Destroyer's Destruction Beam weapon does full damage to secondary targets.",
     ItemNames.WARP_PRISM_GRAVITIC_DRIVE: "Increases the movement speed of Warp Prisms.",
     ItemNames.WARP_PRISM_PHASE_BLASTER: "Equips Warp Prisms with an auto-attack that can hit ground and air targets.",
-    ItemNames.WARP_PRISM_WAR_CONFIGURATION: None,
-    ItemNames.OBSERVER_GRAVITIC_BOOSTERS: None,
-    ItemNames.OBSERVER_SENSOR_ARRAY: None,
-    ItemNames.REAVER_SCARAB_DAMAGE: None,
-    ItemNames.REAVER_SOLARITE_PAYLOAD: None,
-    ItemNames.REAVER_REAVER_CAPACITY: None,
+    ItemNames.WARP_PRISM_WAR_CONFIGURATION: "Warp Prisms transform faster and gain increased power radius in Phasing Mode.",
+    ItemNames.OBSERVER_GRAVITIC_BOOSTERS: "Increases Observer movement speed.",
+    ItemNames.OBSERVER_SENSOR_ARRAY: "Increases Observer sight range.",
+    ItemNames.REAVER_SCARAB_DAMAGE: "Reaver Scarabs deal +25 damage.",
+    ItemNames.REAVER_SOLARITE_PAYLOAD: "Reaver Scarabs gain increased splash damage radius.",
+    ItemNames.REAVER_REAVER_CAPACITY: "Reavers can store 10 Scarabs.",
     ItemNames.REAVER_RESOURCE_EFFICIENCY: get_resource_efficiency_desc(ItemNames.REAVER),
-    ItemNames.VANGUARD_AGONY_LAUNCHERS: None,
-    ItemNames.VANGUARD_MATTER_DISPERSION: None,
-    ItemNames.IMMORTAL_ANNIHILATOR_SINGULARITY_CHARGE: None,
-    ItemNames.IMMORTAL_ANNIHILATOR_ADVANCED_TARGETING_MECHANICS: None,
-    ItemNames.COLOSSUS_PACIFICATION_PROTOCOL: None,
-    ItemNames.WRATHWALKER_RAPID_POWER_CYCLING: None,
-    ItemNames.WRATHWALKER_EYE_OF_WRATH: None,
-    ItemNames.DARK_TEMPLAR_AVENGER_BLOOD_HUNTER_SHROUD_OF_ADUN: None,
-    ItemNames.DARK_TEMPLAR_AVENGER_BLOOD_HUNTER_SHADOW_GUARD_TRAINING: None,
-    ItemNames.DARK_TEMPLAR_AVENGER_BLOOD_HUNTER_BLINK: None,
+    ItemNames.VANGUARD_AGONY_LAUNCHERS: "Increases Vanguard attack range by 2.",
+    ItemNames.VANGUARD_MATTER_DISPERSION: "Increases Vanguard attack area.",
+    ItemNames.IMMORTAL_ANNIHILATOR_SINGULARITY_CHARGE: "Increases Immortal and Annihilator attack range by 2.",
+    ItemNames.IMMORTAL_ANNIHILATOR_ADVANCED_TARGETING_MECHANICS: "Immortals and Annihilators can attack air units.",
+    ItemNames.COLOSSUS_PACIFICATION_PROTOCOL: "Increases Colossus attack speed.",
+    ItemNames.WRATHWALKER_RAPID_POWER_CYCLING: "Reduces the charging time and increases attack speed of the Wrathwalker's Charged Blast.",
+    ItemNames.WRATHWALKER_EYE_OF_WRATH: "Increases Wrathwalker weapon range by 1.",
+    ItemNames.DARK_TEMPLAR_AVENGER_BLOOD_HUNTER_SHROUD_OF_ADUN: f"Increases {DISPLAY_NAME_CLOAKED_ASSASSIN} maximum shields by 80.",
+    ItemNames.DARK_TEMPLAR_AVENGER_BLOOD_HUNTER_SHADOW_GUARD_TRAINING: f"Increases {DISPLAY_NAME_CLOAKED_ASSASSIN} maximum life by 40.",
+    ItemNames.DARK_TEMPLAR_AVENGER_BLOOD_HUNTER_BLINK: "Dark Templar, Avengers, and Blood Hunters unlock Blink.",

```

Comment:
This is a bit inconsistent with the other descriptions; it would read better if it were
"Dark Templar, Avengers, and Blood Hunters gain the Blink ability."
Like Hallucination.